connexion/middleware/request_validation.py

```diff
@@ -113,9 +112,9 @@ async def __call__(self, scope: Scope, receive: Receive, send: Send):
                         self._operation.parameters, self._operation.body_definition()
                     ),
                 )
-                receive_fn = await validator.wrapped_receive()
+                next_app = validator
```

Comment:
Validators are now ASGI apps. Turtles all the way down :)

Reply:
Actually, creating the `AbstractResponseBodyValidator` class, I remembered that the response validators cannot be ASGI Apps, as they can only be created at response time. So reverted this to keep the different validators aligned.